bodhi/server/validators.py

```diff
@@ -800,48 +778,60 @@ def validate_update(request, **kwargs):
         request.errors.status = HTTPNotFound.code
 
 
-def validate_update_owner(request, **kwargs):
+def common_validate(param, request):
```

Comment:
Let's give this a more descriptive name, perhaps something like "```ensure_user_exists()```.

Reply:
Nice, make sense!